index.html

```diff
@@ -602,9 +602,9 @@ <h2>Our Team</h2>
             <div class="col-sm-6 col-md-3 team-member">
               <img src="img/team/noam.jpg">
               <br>
-              <!-- <a href="#"><img src="img/icons/telegram.png"></a> -->
+              <a href="@Copel"><img src="img/icons/telegram.png"></a>
```

Comment:
All of the telegram links need to be fixed to actual links.
I believe the structure would be: `https://t.me/Copel`